src/pages/linux/How-to-install-Java-development-Kit-8-in-Ubuntu-Linux/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: How to install Java development Kit (JDK) 8 in Ubuntu Linux
+---
+
+# How to install Java development Kit (JDK) 8 in Ubuntu Linux
+
+
+You’ve probably decided to learn [Java](https://en.wikipedia.org/wiki/Java_(programming_language)) programming language, or want to test programming Java applications on a Linux platform. But before your write any code in Java you must fist get you tools together. 
+
+Aside from a Java text editor or integrated development environment (IDE), and the most important one the Java development kit (JDK). The JDK is essential for developing and testing Java programs.
+
+
+In this post you'd would be installing the latest version of the JDK (jdk 8). Lets Go!
+
+### STEP 1 – Downloading Java.
+
+
```

Comment:
Since this is a guide for beginners, I believe it would be best to show how to install Java using apt. I'm thinking something like this can replace everything up to step 3:

`First, update the package database so that you'll install the latest version of Java by entering the following into the command terminal:`

```linux
sudo apt update
```

`Then, run the following command to install the JDK:`

```linux
sudo apt install default-jdk
```

Reply:
Sorry, the code blocks aren't showing up properly here, but I'm sure you get the point.